frontend/www/js/omegaup/components/user/Profile.test.ts

```diff
@@ -0,0 +1,41 @@
+import { shallowMount } from '@vue/test-utils';
+import { types } from '../../api_types';
+import expect from 'expect';
+import user_Profile from './Profile.vue';
+
+describe('Profile.vue', () => {
+  it('Should display Profile', () => {
+    const badge_alias = 'contestManager';
+    const badges = ['100SolvedProblems'];
+    const wrapper = shallowMount(user_Profile, {
+      propsData: {
```

Comment:
en las pruebas siempre hay que ponerle la anotación de los tipos a los props, porque de otra manera TypeScript no puede saber qué tipo debe ser:

```suggestion
      propsData:  <types.UserProfileDetailsPayload>{
```